test/e2e/app-dir/actions-allowed-origins/app-action-allowed-origins.test.ts

```diff
@@ -12,7 +12,7 @@ describe('app-dir action allowed origins', () => {
       'server-only': 'latest',
     },
     // An arbitrary & random port.
-    forcedPort: '41831',
+    forcedPort: 'random',
```

Comment:
how come we don't call `findPort()` in the test and then just pass that port as an env var to `nextTestSetup` so it can be used in `next.config`?

Reply:
It's because `describe()` has to be sync, it cannot be `async`, we cannot call the `await findPort()` here directly. It has to be encapsulated into `nextTestSetup`. `forcePort: "random"` doesn't sound ideal, we could find some better name to polish it later. Open to any good names